commit fe45d10456ec57256b86b44fa9a8e0e4d1cf4f4a (refs/changes/52/1552/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-02-22T16:55:42+01:00 (2 years, 7 months ago)
    
    TOOLS-1699 sdcadm health -j should always provide JSON even on exceptions

diff --git a/lib/cli/index.js b/lib/cli/index.js
index 4a8f82b..2e426ed 100644
--- a/lib/cli/index.js
+++ b/lib/cli/index.js
@@ -182,7 +182,6 @@ CLI.prototype.init = function init(opts, args, callback) {
             callback();
             return;
         }
-
         self.sdcadm = new SdcAdm({
             log: self.log,
             uuid: self.uuid
diff --git a/lib/sdcadm.js b/lib/sdcadm.js
index 7364ce0..e7e2a74 100644
--- a/lib/sdcadm.js
+++ b/lib/sdcadm.js
@@ -362,6 +362,9 @@ SdcAdm.prototype.init = function init(cb) {
 
         self.getApp({app: 'sdc'}, function (appErr, app) {
             if (appErr) {
+                // Trent: what should be the better approach to output JSON
+                // messages here? Add a top level '-j|--json' option to
+                // sdcadm? maybe something else?
                 // Couple known issues we can help operators with a friendly
                 // message instead of the default "ENO..." errors:
                 if (appErr.message) {
